.env_sample

```diff
@@ -39,6 +39,19 @@ export OPENRESTY_URL=""
 # SSL config
 export FORCE_SSL_IN_RAILS="not applicable in dev"
 
+# Postgres DB
+export DATABASE_NAME="Forem_development" # Used when creating a db instance `rails db:create`
+export DATABASE_URL="postgresql://forem:forem@localhost:5432/$DATABASE_NAME"
```

Comment:
These require the `forem` role to be created, which is not true by default, `rails db:create` doesn't create roles.

and by default you get:

```shell
psql: error: FATAL:  role "forem" does not exist
```

I think we should remove the `username:password` combo

I think we should also make it explicit that `DATABASE_URL` will always take precedence on `DATABASE_NAME` if they are mismatched:

```shell
➜  forem git:(master) DATABASE_URL=postgres://localhost/PracticalDeveloper_test rails c -e development
[1] pry(main)> ActiveRecord::Base.connection.current_database
=> "PracticalDeveloper_test"
[2] pry(main)> Rails.configuration.database_configuration["development"]["database"]
=> "PracticalDeveloper_development"
```

as you can see I should be connected to `PracticalDeveloper_development` becase that's what `database:` in the file `database.yml` file says, but instead I'm connected to `PracticalDeveloper_test` because that's what the environment variable says

Reply:
@rhymes I removed the username:password combo. And tried to clear in the .env_sample in one line summarisation.

Should I add more detailed documentation as apart of this PR? Maybe in config-env.md